src/locales/scripts/utils.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Mutates an object at the path with the value. If the path
+ * does not exist, it is created by nesting objects along the
+ * path segments.
+ *
+ * From https://stackoverflow.com/a/20240290
```

Comment:
We can use `@see` and `@link`:
```suggestion
 * @see {@link https://stackoverflow.com/a/20240290|Stack Overflow}
```